<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>library_auto_scan_widget</class>
 <widget class="QWidget" name="library_auto_scan_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>常规库扫描</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="5" column="0">
      <widget class="QTextEdit" name="lib_dir_edit_dbg"/>
     </item>
     <item row="7" column="0">
      <widget class="QTextEdit" name="libs_edit_dbg"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>附加包含目录（Debug）</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>附加依赖项（Debug）</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTextEdit" name="incs_edit_dbg"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>附加库目录（Debug）</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>附加包含目录（Release）</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1,2,1,1,0">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Library</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="lib_combo"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Platform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="platform_combo"/>
       </item>
       <item>
        <widget class="QPushButton" name="scan_btn">
         <property name="text">
          <string>扫描</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ok_btn">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="incs_edit_rel"/>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="lib_dir_edit_rel"/>
     </item>
     <item row="7" column="1">
      <widget class="QTextEdit" name="libs_edit_rel"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>附加依赖项（Release）</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>附加库目录（Release）</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
